shared/firestore-models/util.ts

```diff
@@ -0,0 +1,27 @@
+import * as firebase from 'firebase-admin';
+
+import {getConfigRef, BranchManagerRepoConfig} from './repo-config'
+import {BranchManagerPullRequest} from './pull-request'
+
+
+export async function isTargetLabel(configId: string, label: string) {
+  const configRef = await getConfigRef(configId);
+  const configDoc = await configRef.get();
+  if (configDoc.exists) {
+    const config = configDoc.data() as BranchManagerRepoConfig;
+    return !!config.branches.find(branch => branch.label === label);
+  }
+  return false;
+}
+
+export async function hasTargetedLabel(configId: string, pullRequestRef: firebase.firestore.DocumentReference) {
+  const configRef = await getConfigRef(configId);
+  const configDoc = await configRef.get();
```

Comment:
We often seem to call `getConfigRef` and then just await the actual snapshot. Should we just make it a helper function within `repo-config.ts`?

Reply:
I initially made a helper for doing that, but it ended up that we had to `getConfigDoc` calls `getConfigRef` so if you want to get both values, you end up calling `getConfigRef` twice and having to wait on both promises.

I am not sure how to handle it to be efficient

Reply:
`getConfigDoc` could also accept a reference optionally, so if you want both values, you just pass the reference to the `getConfigDoc` function.

Reply:
Updated as I was misreading what was being talked about here.